testing/utils/async_.py

```diff
@@ -0,0 +1,21 @@
+"""
+Provide utils for async functions handling.
+"""
+
+
+async def return_async_value(value):
```

Comment:
Proposal to do `_` prefix before module name like `_async`

Reply:
@DeV1doR, done.